app/src/main/res/layout/activity_nfc.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/root"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".nfc.NfcActivity">
+
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:elevation="4dp"
+        android:theme="@style/ThemeOverlay.HomeAssistant.ActionBar" />
+
+    <ProgressBar
+        android:id="@+id/nfc_loading"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="100dp" />
```

Comment:
I hope the activity doesn't take that long to create that we can see this loading bar?

Reply:
The progressbar is shown when an nfc tag is scanned and the request is send to the server 🙈 The activity is faster than that! 😅